### PR TITLE
Revamp Getting Started Articles

### PR DESCRIPTION
**Pull request by yordan-mitev**
- Remove Visual Studio for Mac Getting Started articles
- Remove PKG installation Getting Started article
- Revamp Getting Started with VS
- Add Getting Started with VC Code

### CODE REVIEW

File: get-started/first-steps-vs.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+
+## Step 4: Add the Telerik Namespace and Register the Controls
+
+To use the controls in the Telerik UI for .NET MAUI library, add the Telerik namespace:
+
+1. In the `MainPage.xaml` file, locate the root element at the top. with the already available namespaces.
+1. Paste the Telerik namespace below the last already available namespace:
+
+	```
+	xmlns:telerikCombo="clr-namespace:Telerik.Maui.Controls;assembly=Telerik.Maui.Controls"
```
Comment (kerpecheva) [inline get-started/first-steps-vs.md:101]: xmlns:telerik="http://schemas.telerik.com/2022/xaml/maui"

File: get-started/first-steps-vs.md [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+
+	![Telerik UI for .NET MAUI - create new MAUI project in Visual Studio](./images/gs-vs-create-maui-app.png)
+	
+1. Name your project and select a location.
+
+1. Choose the .NET framework for your project.
+
+1. Wait until Visual Studio restores all dependencies (when done, all exclamation marks in the **Dependencies** tree view item disappear).
+
+1. Click the **Windows Machine** button to build and run the app.
```
Comment (kerpecheva) [inline get-started/first-steps-vs.md:50]: They can run it on Android as well, so here we can just say Run the app

Comment (yordan-mitev) [reply get-started/first-steps-vs.md:50]: > They can run it on Android as well, so here we can just say Run the app

Couldn't this lead to some implications? Suppose the developer doesn't have the required Android emulator, this means that we need to add several additional steps.

File: get-started/first-steps-vs.md (hunk 1 above)
Comment (yordan-mitev) [inline get-started/first-steps-vs.md:101]:
```suggestion
	xmlns:telerik="http://schemas.telerik.com/2022/xaml/maui"
```

**Review by didiyordanova — APPROVED** [review]

File: get-started/first-steps-vs.md (hunk 2 above)
Comment (didiyordanova) [reply get-started/first-steps-vs.md:40]: I am okay to keep the windows platform as a getting started scenario for VS. Yes having android the user has to be sure the emulators are setup. this is additional step.

**Review by ikoevska — COMMENTED** [review]
Ignored.

**Pull request merged**